llvm/test/tools/repo2obj/extern_mixed_weak_global_references.ll

```diff
@@ -0,0 +1,56 @@
+// Check the weak symbol binding: the output symbol is global if one or more of the references to it are strong.
+//
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: pstore-import %t/db.db %s
+// RUN: repo-create-ticket --repo %t/db.db -o %t/t.o e865ac76298a3007c24cad11885532d4
+// RUN: repo2obj %t/t.o --repo=%t/db.db -o %t/t.elf
+// RUN: llvm-readobj --symbols %t/t.elf | FileCheck %s
+
+// CHECK:    Name: _ZTH1a (8)
+// CHECK-NEXT:    Value: 0x0
+// CHECK-NEXT:    Size: 0
+// CHECK-NEXT:    Binding: Global (0x1)
+// CHECK-NEXT:    Type: None (0x0)
+// CHECK-NEXT:    Other: 0
+// CHECK-NEXT:    Section: Undefined (0x0)
+
+{
+  "version":1,
+  "id":"ea3cc7c1-c2a6-4d91-a0c3-eae84ba60d12",
+  "transactions":[
+    {
+      // generation 1
+      "names":[
+        "x86_64-pc-linux-gnu-repo",
+        "\\\\?\\C:\\mywork\\github\\llvm-project-prepo\\build\\test\\tools\\repo2obj\\Output",
+        "a",
+        "_ZTW1a",
+        "_ZTH1a"
+      ],
+      "debugline":{
+      },
```

Comment:
The empty ”debugline” property can be safely removed. (I want to rename it "debug-line-header" to reflect its true role and including it here will result in a test failure when that happens.)

Reply:
Removed.